packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardFilterBuilder.tsx

```diff
@@ -55,6 +55,16 @@ export interface CloudPulseDashboardFilterBuilderProps {
 
   handleToggleAppliedFilter: (isVisible: boolean) => void;
 
+  /**
+   * Is cluster Call
+   */
+  isResourceCallError?: boolean;
+
+  /**
+   * Property to disable filters
+   */
+  isResourceCallLoading?: boolean;
+
```

Comment:
```suggestion
  /**
   * Is cluster Call
   */
  isError?: boolean;

  /**
   * Property to disable filters
   */
  isLoading?: boolean;

```

these common property names will be much better as we don't have any other loading or error state in the component

Reply:
sure